roles/ansible_service_broker/defaults/main.yml

```diff
@@ -0,0 +1,7 @@
+---
+
+ansible_service_broker_remove: false
+ansible_service_broker_namespace: openshift-ansible-service-broker
```

Comment:
after talking with @pmorie yesterday regarding our namespaces, i dont think we'll want customers to change them at this time so we actually should just hard-code `openshift-ansible-service-broker`. I recently made the change in my own branch...